install.ps1

```diff
@@ -0,0 +1,74 @@
+function getTitleFromBestPracticesFile {
+    $titleMatchRaw = cat /PSScriptAnalyzer/PowerShellBestPractices.md | grep $args[0] ;
+    if($titleMatchRaw) { $titleMatchRaw.split('[')[0].SubString(2) } ;
+}
+
+function getTitleFromRuleFile {
+    $patternNameCamelCased = $args[0] ;
+    #--- get title from finding rule files that have a name that matches part of a pattern name
+    foreach($file in (ls /PSScriptAnalyzer/Rules/).split('\n')) {
+        if($patternNameCamelCased.contains($file.split('.cs')[0])){
+            $ruleFileWithShortPatternName = '/PSScriptAnalyzer/Rules/' + $file;
+            $shortPatternName = $file.split('.cs')[0] ;
+        }
+    } ;
+    $grepPatternShortName = '/// ' + $shortPatternName + ': ' ;
+    # the next instruction will sometimes output 'No such file or directory' from cat
+    $titleFromShortNameGrepResult = cat $ruleFileWithShortPatternName | grep "$grepPatternShortName" ;
```

Comment:
Is this better than doing an `if($ruleFileWithShortPatternName) { $ruleFileWithShortPatternName }`?